Subscription.cs

```diff
@@ -42,7 +42,7 @@ public class Subscription : IDisposable
         public LatentSequence<AvailableExtensionImage> ExtensionImages { get; private set; }
         public LatentSequence<ReservedIp> ReservedIps { get; private set; }
 
-        HttpClient _coreHttpClient20140601, _coreHttpClient20141001, _databaseHttpClient;
+        HttpClient _coreHttpClient20140601, _coreHttpClient20141001,_coreHttpClient20120301, _databaseHttpClient;
```

Comment:
New code should be using the latest version (20141001 last I checked) not adding a client that uses an older version.